packages/nuxt/src/app/composables/route-announcer.ts

```diff
@@ -0,0 +1,94 @@
+import type { Ref } from 'vue'
+import { getCurrentScope, onScopeDispose, ref } from 'vue'
+import { getActiveHead } from 'unhead'
+import { useNuxtApp } from '#app'
+
+export enum Politeness {
+  Assertive = 'assertive',
+  Polite = 'polite',
+  Off = 'off',
+}
+
+export type PolitenessValue = `${Politeness}`
+
+export type NuxtRouteAnnouncerOpts = {
+  /** @default 'polite' */
+  politeness?: PolitenessValue
+}
+
+export type RouteAnnouncer = {
+  message: Ref<string>
+  politeness: Ref<PolitenessValue>
+  set: (message: string, politeness: PolitenessValue) => void
+  polite: (message: string) => void
+  assertive: (message: string) => void
+  _cleanup: () => void
+}
+
+function createRouteAnnouncer (opts: NuxtRouteAnnouncerOpts = {}) {
+  const message = ref('')
+  const politeness = ref<PolitenessValue>(opts.politeness || Politeness.Polite)
+  const activeHead = getActiveHead()
```

Comment:
This needs to come from @unhead/vue to work outside of a Vue context. `injectHead` is the function

Reply:
Thanks for pointing this out! I've changed the code to use `injectHead` from `@unhead/vue `